.circleci/config.yml

```diff
@@ -427,8 +875,134 @@ workflows:
+      - back-ee-trigger-deploy:
+          context: circleci-api-token
+          requires:
+            - back-ee-test
+            - back-ee-slow-tests
+          filters:
+            branches:
+              only:
+                - master
+                - production
+
+  back-ee-deploy:
+    when:
+      and:
+        - <<pipeline.parameters.run_deploy>>
+        - << pipeline.parameters.back >>
+    jobs:
+      - back-ee-push-deployment-docker-image:
+          context: docker-hub-access
+          filters:
+            branches:
+              only:
+                - master
+                - production
+      - back-ee-deploy-to-swarm:
+          requires:
+            - back-ee-push-deployment-docker-image
+          filters:
+            branches:
+              only:
+                - master
+          ssh_host: $CITIZENLAB_CLUSTER_IP_ADDRESS_STAGING
+          compose_file: docker-compose-staging.yml
+          stack_name: cl2-back-stg
+          env_file: ".env-staging"
+      - back-ee-deploy-to-swarm:
+          requires:
+            - back-ee-push-deployment-docker-image
+          filters:
+            branches:
+              only:
+                - production
+          ssh_host: $CITIZENLAB_CLUSTER_IP_ADDRESS_EUROPE
+          compose_file: docker-compose-production-benelux.yml
+          stack_name: cl2-prd-bnlx-stack
+          env_file: ".env-production-benelux"
+      - back-ee-deploy-to-swarm:
+          requires:
+            - back-ee-push-deployment-docker-image
+          filters:
+            branches:
+              only:
+                - production
+          ssh_host: $CITIZENLAB_CLUSTER_IP_ADDRESS_CANADA
+          compose_file: docker-compose-production-canada.yml
+          env_file: ".env-production-canada"
+      - back-ee-deploy-to-swarm:
+          requires:
+            - back-ee-push-deployment-docker-image
+          filters:
+            branches:
+              only:
+                - production
+          ssh_host: $CITIZENLAB_CLUSTER_IP_ADDRESS_SAM
+          compose_file: docker-compose-production.yml
+          env_file: ".env-web"
+          stack_name: "cl2"
+      - back-ee-deploy-to-swarm:
+          requires:
+            - back-ee-push-deployment-docker-image
+          filters:
+            branches:
+              only:
+                - production
+          ssh_host: $CITIZENLAB_CLUSTER_IP_ADDRESS_USW
+          compose_file: docker-compose-production.yml
+          env_file: ".env-web"
+          stack_name: "cl2"
+      - back-ee-deploy-to-swarm:
+          requires:
+            - back-ee-push-deployment-docker-image
+          filters:
+            branches:
+              only:
+                - production
+          ssh_host: $CITIZENLAB_CLUSTER_IP_ADDRESS_UK
+          compose_file: docker-compose-production.yml
+          env_file: ".env-web"
+          stack_name: "cl2"
+
+  # FRONT-END ESSENTIAL
+  front-essential:
     when: << pipeline.parameters.front >>
```

Comment:
Same here, I suspect we need to check `run_deploy` var too here, or it'll re-run when deploying